resources/home_page.htt

```diff
@@ -47,6 +47,7 @@
                 document.getElementById('bal_value').innerHTML=grand_json['BALVALUE'];
             </script>
         </td>
+        <tr height='25px'></tr>
```

Comment:
Does this work correctly if the font size is bigger?

![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/dad60ffe-e2f3-4353-84c3-eef63fe4ab58)

Thank you